.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
```

Comment:
In the future, if you want your eslint configuration to work for Jest as well, there will be a few other things that have to be added in here as well!